src/webui/api/torrentscontroller.cpp

```diff
@@ -403,7 +403,7 @@ void TorrentsController::trackersAction()
             {KEY_TRACKER_TIER, tracker.tier()},
             {KEY_TRACKER_STATUS, static_cast<int>(tracker.status())},
             {KEY_TRACKER_PEERS_COUNT, data.numPeers},
-            {KEY_TRACKER_MSG, data.lastMessage.trimmed()},
+            {KEY_TRACKER_MSG, data.messages.join(" | ")},
```

Comment:
Would it be a bad idea to add an additional `trimmed()` in the end? Just to keep it equivalent to what it was.

Reply:
It is trimmed when adding message. https://github.com/qbittorrent/qBittorrent/pull/12173/files#diff-aad6818285ca77451681edce2d6761a6R62